application/clicommands/ListCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Icinga\Module\Reporting\Clicommands;
+
+use Icinga\Module\Reporting\Cli\Command;
+use InvalidArgumentException;
+use ipl\Sql\Select;
+
+class ListCommand extends Command
+{
+    /**
+     * * USAGE:
```

Comment:
You've replaced `List reports` as well. Please add it again and remove the redundant `* ` before `USAGE`.